pages/common/laravel.md

```diff
@@ -0,0 +1,15 @@
+# laravel
+
+> The PHP Framework For Web Artisans.
+
+- Create a new Laravel application:
+
+`new`
```

Comment:
A novice copying-and-pasting this into the command line won't see the expected result here. Perhaps these could be expanded to be the actual commands you would execute would help? For example, on the `adduser` page, we have the example `adduser --no-create-home {{username}}`.

Reply:
Yeh, this is `laravel new {{name}}`.

There is also `laravel new {{name}} --force`. :+1: